### ISSUE TITLE
Information page on MassBank

### ISSUE THREAD

**Issue by tsufz** [MassBank/MassBank-web#228]
Hi, I suggest to add an information page on MassBank. The content could be for example the connection with our partners, a link to RMassBank etc. 

This can make MassBank more visible:

If everyone is sending some information, the compilation is easy. Volunteers?

I can do for example the MZmine things, 

I would like to clear the news section out of such hidden functionalities.

Best,
Tobi

**Comment by schymane** [comment]
It's a great idea, but it may be easier if you make an outline of what you are envisaging and then assign (suggest) people to contribute certain content bits. It might be easier than waiting for people to send contributions (my immediate question is "what"). I can imagine we won't want a book or essay ... but I cannot quite see how you are intending to display this.

**Comment by schymane** [comment]
I agree that we really need an "about" site, here's a starter snippet that I will send to Tony for the non-target analysis website that is being set up apparently ... maybe this could be a good starter point (modified from the latest NORMAN bulletin). 


The European MassBank (https://massbank.eu/) is a member of the MassBank Consortium (https://github.com/MassBank) alongside the founding MassBank of Japan (http://massbank.jp/) and MassBank of North America (https://massbank.us). The aim of the European MassBank is to provide an open access and vendor-independent repository for mass spectral data, to support the screening and identification  of  unknown  compounds, with a particular focus on environmental samples due to members of the European community. Due to efforts in Europe, MassBank.EU now contains >88,000 spectra of >16,500 compounds from >40 instrument/ionisation types and >45 contributors. As such, MassBank is an essential contribution to improved identification of hazardous compounds in Europe and worldwide. In line with this objective, MassBank improves the flexible exchange of information within several European and NORMAN (https://www.norman-network.com/) projects, as well as many other activities. MassBank.EU is operated by the Helmholtz Centre for Environmental Research - UFZ in Leipzig, Germany (https://www.ufz.de/index.php?en=33573) and co-maintaned by the IPB Halle (https://www.ipb-halle.de/en/) and LCSB, University of Luxembourg (https://wwwen.uni.lu/lcsb/).
